packages/google_maps_flutter/google_maps_flutter/example/lib/main.dart

```diff
@@ -34,6 +35,7 @@ final List<GoogleMapExampleAppPage> _allPages = <GoogleMapExampleAppPage>[
   PlacePolylinePage(),
   PlacePolygonPage(),
   PlaceCirclePage(),
+  DragMarkerPage(),
```

Comment:
There's another demo that deals with dragging markers, maybe this one can be merged into the PlaceMarkerPage (or the dragging demo of the PlaceMarkerPage can be moved to this one)